package-lock.json

```diff
@@ -1,6843 +1,8 @@
 {
   "name": "stytch",
-  "version": "3.5.1",
-  "lockfileVersion": 2,
+  "version": "3.6.0",
+  "lockfileVersion": 1,
```

Comment:
I think this is an NPM version mismatch, since `lockfileVersion: 2` was added in NPM v7. My local version number, if it helps:
```
$ npm --version
7.24.0
```

Reply:
Ah yes, I was running 6 something. Updated and regenerated, most of changes to package-lock ended up reverted, 8148e73